src/index.html

```diff
@@ -10,23 +10,35 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="form"
+    >
+      <input
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles“"
```

Comment:
The placeholder text contains a space within the quotes. Ensure that links or anchor tags do not contain spaces, as per the checklist.